README.md

```diff
@@ -26,13 +28,22 @@ import { RPCHandler, HandlerConstructorConfig } from "@ubiquity-dao/rpc-handler/
 
 export function useHandler(networkId: number) {
   const config: HandlerConstructorConfig = {
-    networkId: 1,
-    rpcTimeout: 1500,
-    autoStorage: false,
-    cacheRefreshCycles: 10,
-    networkName: null, // the name will be deduced from the networkId, unless using a custom network
-    networkRpcs: null, // same as networkName, but for injecting additional RPCs
-    runtimeRpcs: null, // same as networkRpcs, although these are considered error-free
+    networkId: 1;
```

Comment:
should we default to Ethereum Mainnet? why?

Reply:
It's more just a point of reference than a default but I will convert it into a cut and paste default config. I copied and pasted half of a literal config and half of a typed config here I just noticed too.